gh-badges/templates/_shields_test-template.json

```diff
@@ -1,9 +1,7 @@
 {
-{{?it.color}}
-  "color": {{=JSON.stringify(it.color)}},
-{{?}}
+  "color": {{=JSON.stringify(it.color || null)}},
```

Comment:
This restores `color` always be present in this object, for parity with the old behavior.

If we implement #2687 we can revisit this format. The behavior I mentioned in https://github.com/badges/shields/pull/2780#issuecomment-454879950 was very unexpected and it would be nice if we could depend on this more.